src/servers/src/http/event.rs

```diff
@@ -612,10 +612,30 @@ async fn ingest_logs_inner(
     let mut results = Vec::with_capacity(pipeline_data.len());
     let transformed_data: Rows;
     if pipeline_name == GREPTIME_INTERNAL_IDENTITY_PIPELINE_NAME {
-        let rows = pipeline::identity_pipeline(pipeline_data)
+        let (mut schema_info, rows) = pipeline::identity_pipeline(pipeline_data)
             .context(PipelineTransformSnafu)
             .context(PipelineSnafu)?;
-        transformed_data = rows;
+        let catalog = query_ctx.current_catalog();
+        let schema = query_ctx.current_schema();
+        let table = state
+            .get_table(catalog, &schema, &table_name)
+            // FIXME: handle error
+            .await
+            .context(CatalogSnafu)?;
+
+        if let Some(table) = table {
+            let schema = &table.table_info().meta;
+            schema.row_key_column_names().for_each(|column_name| {
```

Comment:
I think the name of the function `row_key_column_names` is not clear enough.
Shall we change it to other like `tag_column_names`.